composer.json

```diff
@@ -34,6 +34,7 @@
         "symfony/config": "^4.3.3|^5.0|^6.0",
         "symfony/console": "^3.4.30|^4.3.3|^5.0|^6.0",
         "symfony/dependency-injection": "^4.3.3|^5.0|^6.0",
+        "symfony/deprecation-contracts": "^2.1",
```

Comment:
There was no decision about choosing symfony/deprecation-contracts or doctrine/deprecations yet. I would rather not make this decision now.

Reply:
Since this bundle integrates Doctrine into Symfony, I'd expect this Bundle to announce its deprecations using Symfony mechanisms.

Reply:
That's a discusion which should happen in https://github.com/doctrine/DoctrineBundle/issues/1300